aether-common-module/aether/common/conf/settings.py

```diff
@@ -246,12 +303,21 @@
 if DJANGO_STORAGE_BACKEND == 'filesystem':
     DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
     HOSTNAME = os.environ['HOSTNAME']
+    if not HOSTNAME:
+        raise RuntimeError('Missing HOSTNAME environment variable!')
+
 elif DJANGO_STORAGE_BACKEND == 's3':
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
     AWS_STORAGE_BUCKET_NAME = os.environ['BUCKET_NAME']
+    if not AWS_STORAGE_BUCKET_NAME:
+        logger.critical('Missing BUCKET_NAME environment variable!')
+
 elif DJANGO_STORAGE_BACKEND == 'gcs':
     DEFAULT_FILE_STORAGE = 'storages.backends.gcloud.GoogleCloudStorage'
     GS_BUCKET_NAME = os.environ['BUCKET_NAME']
+    if not GS_BUCKET_NAME:
+        logger.critical('Missing BUCKET_NAME environment variable!')
```

Comment:
@adamvert and @MrPink what do you prefer? To raise a `RuntimeError` or to send the critical log to the console/sentry?

Reply:
Can we do both?

Reply:
Always!